SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DefaultApplicationSettings.cs

```diff
@@ -51,33 +67,55 @@ public bool ContainsKey(string key)
         {
             lock (configLock)
             {
-                return configuration.AppSettings.Settings[key] != null;
+                if (configuration != null)
+                {
+                    return configuration.AppSettings.Settings[key] != null;
```

Comment:
Will this work?
```suggestion
                    return configuration?.AppSettings?.Settings[key] != null;
```

Reply:
Not entirely sure though

Reply:
I don't use 'configuration?' because I want to log the case when 'configuration' is null.